package.json

```diff
@@ -112,16 +112,5 @@
     "extension": true,
     "outputDir": "jupyterlab_universal_extension/labextension",
     "schemaDir": "schema"
-  },
-  "jupyter-releaser": {
-    "hooks": {
-      "before-build-npm": [
-        "python -m pip install jupyterlab~=3.1",
-        "jlpm"
-      ],
-      "before-build-python": [
-        "jlpm clean:all"
-      ]
-    }
```

Comment:
This is an old heritage of the project template. It should now be specified in `pyproject.toml`